### ISSUE TITLE
"opengraph: false" erroring

### ISSUE THREAD

**Issue by haydenbleasel** [haydenbleasel/favicons#82]
Originally: https://github.com/haydenbleasel/gulp-favicons/issues/23

> Hi,
> Everything is now working beautifully, the only slight issue is when I disable the Open Graph image (which I need to handle independently to get the article image rather than a favicon image), but setting opengraph: false in the icons parameters causes the following error: 

```
C:\Autumn Earth\Dropbox\autumn-earth\site\node_modules\favicons\helpers.js:253 Reflect.deleteProperty(request.favicon_design, 'open_g ^ ReferenceError: Reflect is not defined at Object.configure (C:\Autumn Earth\Dropbox\autumn-earth\site\node_modules\favicons\helpers.js:253:25)
```

> Any idea? I can set the Twitter card image to false without any problem - it just seems to be the Open Graph that causes the issue.

...

> I have run into the same issue. It only occurs when online is set to true. Any set of icons can be set to false, not just opengraph for the issue to occur. Running v5.1.1 of node.

CC: @johnholtripley @IanCaunce

**Comment by haydenbleasel** [comment]
Update: This happens by setting any icon to `false` with `online: true`. It spits out `Reflect is not defined` which is strange as we're running ES6 all over the place. Potentially related issue:

> The cause is that after compilation step browserify wraps every source file into:

```
module.exports = function() {
  "use strict";
  (source file)
};
```
> In other words it forces to use strict mode where this is undefined. To solve the issue I just used

```
(function() {
  var global = this || window;
  ...
})(); 
```

**Comment by haydenbleasel** [comment]
Added `harmony-reflect` polyfill, should be fixed in `favicons@4.3.1`.
